### DISCUSSION TITLE
Issue while creating job position

### DISCUSSION THREAD

**Discussion by sharukhkhan234** [horilla-opensource/horilla#257] [Q&A]
@horilla-opensource !
As you have provided a new code for the job position, I did it the same, but again facing an issue, not showing any error nor creating a job position, it just showed a blank white screen which you can see below, please provide an error-free solution for it.

![err0r1](https://github.com/user-attachments/assets/b52026fa-f656-45ba-8e32-870e0ad237c4)
![views_1](https://github.com/user-attachments/assets/09a22323-39e5-49d8-b2b0-b95757699dfa)
![views_2](https://github.com/user-attachments/assets/f85e044d-faf0-4ac2-ac34-35e579bff693)

Regards,

**Comment by sharukhkhan234** [comment]
![form](https://github.com/user-attachments/assets/b2059dfa-0b0b-4071-9b31-ebc2813e9a0f)

**Reply by horilla-opensource** [reply]
Hi @sharukhkhan234 ,
If the screen is showing white and no error log in python, please check the browser console. The white screen comes up when you get an error from the ajax requests. 
Please check the console and share the details of the same.

**Reply by sharukhkhan234** [reply]
![job_position](https://github.com/user-attachments/assets/9b29fcc1-16ce-4f73-9fb4-566952dfc71b)

please check this
